media-gfx/zbar/zbar-0.23.ebuild

```diff
@@ -0,0 +1,203 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit autotools flag-o-matic java-pkg-opt-2 multilib-minimal python-single-r1 virtualx
+
+DESCRIPTION="Library and tools for reading barcodes from images or video"
+HOMEPAGE="https://github.com/mchehab/zbar"
+SRC_URI="https://linuxtv.org/downloads/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+IUSE="dbus graphicsmagick gtk gtk2 imagemagick introspection java jpeg png python qt5 static-libs test +threads v4l X xv"
```

Comment:
is there any reason to keep GTK2? Because upstream made it seem like GTK3 is now a full replacement, in which case we do not provide a legacy option if there is no reverse-dependency.

Reply:
this is the library and it can be necessary for another program with gtk2/introspection I think

Reply:
If that were the case, then you switching USE `gtk` to mean GTK3 would already be problematic. But I grepped the tree and there is nothing requesting it.

Reply:
what should I do with this ?😋

Reply:
Drop gtk2 :p